packages/app/src/lib/feathers.ts

```diff
@@ -17,10 +10,6 @@ type Service<T> = AdapterService<T> & ServiceAddons<T>;
 interface ServiceTypes {
   users: Service<User>;
   'nfc-readers': Service<NFCReader>;
```

Comment:
Could you add the `nfc-tags` here as well.

Reply:
fixed in 945a2944136251cfba9c25f265a43eaeaabfad79